### ISSUE TITLE
`@param` javadoc for record classes (JEP 395) are not recognized

### ISSUE THREAD

**Issue by averak** [springdoc/springdoc-openapi#2131]
**Describe the bug**

- I want to use java record classes ([JEP 395](https://openjdk.org/jeps/395)) for schema.
- I use [therapi-runtime-javadoc](https://github.com/dnault/therapi-runtime-javadoc) for generate OAS3 `description` field, but `@param` javadoc is not recognized.
  - If I use normal class, `@param` javadoc is recognized, but record classes aren't recognized.

**To Reproduce**

- What version of spring-boot you are using?
  - 3.0.1
- What modules and versions of springdoc-openapi are you using?
  - `org.springdoc:springdoc-openapi-starter-webmvc-ui:2.0.2`
  - `org.springdoc:springdoc-openapi-starter-common:2.0.2`
  - `com.github.therapi:therapi-runtime-javadoc:0.15.0`
  - `com.github.therapi:therapi-runtime-javadoc-scribe:0.15.0`
- What is the actual and the expected result using OpenAPI Description (yml or json)?
  - `json`
- Provide with a sample code (HelloController) or Test that reproduces the problem

The code below does not work as expected.

```java
/**
 * User
 *
 * @param name name 
 */
public record User(@Schema(requiredMode = Schema.RequiredMode.REQUIRED) name) {}
```

The code below works as expected.

```java
public class User {
    /**
     * name
     */
    @Schema(requiredMode = Schema.RequiredMode.REQUIRED)
    name;
}
```

**Comment by uc4w6c** [comment]
Can I fix it?

**Comment by bnasslahsen** [comment]
@averak,

Patch available in v2.0.4

**Comment by averak** [comment]
@bnasslahsen 

Thank you so much, You've been very helpful!
